src/test.rs

```diff
@@ -15,7 +15,7 @@ use std::process::Command;
 use std::str;
 use std::sync::{Arc, Once, ONCE_INIT};
 
-fn path_to_exe(name: &str) -> PathBuf {
+pub fn path_to_exe(name: &str) -> PathBuf {
```

Comment:
To be able to use it in my tests in `unix.rs` where I do `use test::*;`
Just now when I looked at that again I realized I could also make `true_cmd` public and make use of that.